src/libfetchers/github.cc

```diff
@@ -11,6 +11,7 @@
 #include <optional>
 #include <nlohmann/json.hpp>
 #include <fstream>
+#include <boost/algorithm/string/replace.hpp>
```

Comment:
Why not just `std::string`'s `replace`?

Reply:
I did try using that initially.  
The main problem i hit there was it can only replace a char with another char.  
It does not work when you try to replace a char with a string.